core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -52,22 +57,23 @@ class QueueProcessor(
 
     case RouteeFailed(ex) ⇒
       inflightCreations -= 1
-      log.error(ex, "Failed to retrieve Routee")
+      log.warning("Failed to retrieve Routee: " + ex.getMessage)
```

Comment:
When starting a cluster, this actually happens easily. but it's no longer a blocking error since the `QueueProcessor` will retry and eventually get the routees if the cluster starts up.